File: python/03_primes/primes.py
def isprime(number):
    for cur in range(2, number):
        if number % cur == 0:
            return False
    return True

for number in range(2, 100):
    if isprime(number):
        print(number, "is prime")
    else:
        print(number, "is not prime")

File: rust/03_primes/primes.rs
fn isprime(number:i32) -> bool
{
    for cur in 2..number {
        if number % cur == 0
	{
            return false;
        }
    }
    return true;
}

fn main()
{
    for number in 2..99 {
        if isprime(number){
            println!("{} is prime", number)
        }
	else{
            println!("{} is not prime", number)
        }

    }
}